src/corelibs/U2Designer/src/wizard/WizardController.cpp

```diff
@@ -66,11 +66,11 @@
 
 namespace U2 {
 
-WizardController::WizardController(Schema *s, const Wizard *w)
-    : QObject(), schema(s), wizard(w), runAfterApply(false) {
+WizardController::WizardController(std::shared_ptr<Schema> s, const Wizard *w)
+    : QObject(), schema(move(s)), wizard(w), runAfterApply(false) {
```

Comment:
This thing looks a bit strange (I mean, `std::move`) - a bit over-designed or like you don't get the idea of the move semantic. 

It's difficult to explain why it's not really correct, I've tied to text a good comment, but it was very large and complicated in result. The main idea is: do not use `std::move` in this context, just replace your argument `std::shared_ptr<Schema> s` with `const std::shared_ptr<Schema>& s` (and it's better to use `QSharedPointer`)